simple-guide.md

```diff
@@ -0,0 +1,25 @@
+# How to solve layout tasks 
```

Comment:
There is already Workflow section in README. Can we use it instead?

Reply:
I wan't to simplify the flow 
But the simple flow can be used only after some changes in a task repo
So if I just replace README the instructions will not be relevant to old tasks
So this approach is safe

Reply:
> after some changes in a task repo
How about we make those changes and then update README instead of creating 2 guides, which are almost identical but slightly different?

Reply:
I would like to test the whole flow with one tasks before applying changes to all of them